uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -40,6 +40,7 @@ define([
       this.requestDayTypes = [];
       this.selectedAbsenceType = {};
       this.period = {};
+      this.postContactSelection = false, //flag to track if user is selected for enabling UI
```

Comment:
its not part of loading.x so moved from there as it does not show the spinner, its more to do with disabling of UI which is unrelated to loading